Sources/MapboxMaps/Foundation/BaseMapView.swift

```diff
@@ -127,6 +136,14 @@ open class BaseMapView: UIView {
         if let cameraOptions = resolvedMapInitOptions.cameraOptions {
             mapboxMap.__map.setCameraFor(MapboxCoreMaps.CameraOptions(cameraOptions))
         }
+
+        // Set prefetchZoomDelta
+        let defaultPrefetchZoomDelta: UInt8 = 4
```

Comment:
Similar comment to above, Looks like this same value is being reused so we should make this a private constant in the class

Reply:
There's a ticket open for this. We should address it as part of this.